config/credentials.properties

```
@@ -0,0 +1,7 @@
+XCMAILR_EMAIL = 
+XCMAILR_PASSWORD = 
+EMAIL = 
+EMAIL_PASSWORD = 
+EMAIL_LOGIN = 
+EMAIL_SERVER = 
+SENDER_PORT = 25
```

Remove port and add new line at the end of a file
```suggestion
SENDER_PORT =

```